### PR TITLE
MAR 426 CoCo Dialogflow SDK 

### PR DESCRIPTION
**Pull request by VictorCoCo**

### CODE REVIEW

File: docs/tutorials/plug_coco_component_inside_dialogflow_conversation.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+In the following tutorial we'll implement [CoCo](http://conversationalcomponents.com/ "CoCo")
+component into our [Dialogflow](https://dialogflow.cloud.google.com/ "Dialogflow")
+conversation flow. When a component will be triggered when relevant event matched.
+More specific, we'll use [CoCo](http://conversationalcomponents.com/ "CoCo") [register component](https://marketplace.conversationalcomponents.com/blueprint/register_vp3 "register component")
+from the [CoCo Marketplace](https://marketplace.conversationalcomponents.com/ "CoCo Marketplace").
+
+
+## CoCo SDK Installation To Dialogflow:
+
+**! Setup Your Firebase billing account before starting !**
```
Comment (chenb67) [inline docs/tutorials/plug_coco_component_inside_dialogflow_conversation.md]: firebase billing is only relevant after they deploy the fulfilment function.
and good chance the reader has no idea what is firebase and what he is going to pay for

File: docs/tutorials/plug_coco_component_inside_dialogflow_conversation.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
+# Plug a CoCo Component inside Dialogflow Conversation Flow.
+
+In the following tutorial we'll implement [CoCo](http://conversationalcomponents.com/ "CoCo")
+component into our [Dialogflow](https://dialogflow.cloud.google.com/ "Dialogflow")
+conversation flow. When a component will be triggered when relevant event matched.
```
Comment (chenb67) [inline docs/tutorials/plug_coco_component_inside_dialogflow_conversation.md]: a component will be triggered when a relevant intent matched

File: docs/tutorials/plug_coco_component_inside_dialogflow_conversation.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+from the [CoCo Marketplace](https://marketplace.conversationalcomponents.com/ "CoCo Marketplace").
+
+
+## CoCo SDK Installation To Dialogflow:
+
+**! Setup Your Firebase billing account before starting !**
+
+### Import CoCo Dialogflow SDK functions.
+* Access your agent fulfillment tab:
+![](./screenshots/plug_coco_component_inside_dialogflow_conversation/1_fulfillment_tab.png)
```
Comment (chenb67) [inline docs/tutorials/plug_coco_component_inside_dialogflow_conversation.md]: FYI ./ is redundant in the path

File: docs/tutorials/plug_coco_component_inside_dialogflow_conversation.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+* Enable the inline editor functionality:
+![](./screenshots/plug_coco_component_inside_dialogflow_conversation/2_enable_inline_editor.png)
+* Add CoCo SDK To Package JSON:
+![](./screenshots/plug_coco_component_inside_dialogflow_conversation/3_add_coco_sdk_to_package_json.png)
+
+Copy the line:
+```javascript
+    "@conversationalcomponents/dialogflow-sdk": "^1.0.2"
```
Comment (chenb67) [inline docs/tutorials/plug_coco_component_inside_dialogflow_conversation.md]: this is locking it to future updates and fixes?

File: docs/tutorials/plug_coco_component_inside_dialogflow_conversation.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+```
+* Import CoCo SDK functions:
+![](./screenshots/plug_coco_component_inside_dialogflow_conversation/4_import_coco_dialogflow_sdk.png)
+
+Copy the line:
+```javascript
+const { cocoComponent, cocoContext } = require("@conversationalcomponents/dialogflow-sdk");
+```
+
+### Create Intent For CoCo Context.
```
Comment (chenb67) [inline docs/tutorials/plug_coco_component_inside_dialogflow_conversation.md]: missing the why

Comment (VictorCoCo) [reply docs/tutorials/plug_coco_component_inside_dialogflow_conversation.md]: In order to hold the component context.

File: docs/tutorials/plug_coco_component_inside_dialogflow_conversation.md (hunk 1 above)
Comment (VictorCoCo) [reply docs/tutorials/plug_coco_component_inside_dialogflow_conversation.md]: After fulfilment function

**Pull request merged**